Makefile

```diff
@@ -82,9 +82,12 @@ cover: unit
 	go tool cover -html=coverage.out
 
 e2e:
-	FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]" ./hack/e2e.sh
+	FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]|\[EtcdRecovery\]" ./hack/e2e.sh
```

Comment:
How long does this take? I wouldn't add it in the main suite if the slowdown is huge, instead it can be its own job that runs on every merged commit, and on every PR touching the test itself, the backup, and the restore code.

Reply:
a bit under 30mins, i'll add seperately